src/Uno.UI.Toolkit/MenuFlyoutExtensions.cs

```diff
@@ -0,0 +1,29 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using Windows.UI.Xaml;
+using Windows.UI.Xaml.Controls;
+
+namespace Uno.UI.Toolkit
+{
+#if __IOS__
+	[global::Foundation.PreserveAttribute(AllMembers = true)]
+#elif __ANDROID__
+	[Android.Runtime.PreserveAttribute(AllMembers = true)]
+#endif
+	public class MenuFlyoutExtensions
+	{
+		#region Property: CancelTextIosOverride
+
+		public static DependencyProperty CancelTextIosOverrideProperty { get; } = DependencyProperty.RegisterAttached(
+			"CancelTextIosOverride",
+			typeof(string),
+			typeof(MenuFlyoutExtensions),
+			new PropertyMetadata(default(string)));
+
+		public static string GetCancelTextIosOverride(MenuFlyout obj) => (string)obj.GetValue(CancelTextIosOverrideProperty);
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Refactor this method to add validation of parameter 'obj' before using it.](https://app.codacy.com/gh/unoplatform/uno/pullRequest?prid=5611164)